doc/source/diagnostics/scalability_metrics_plugin.rst

```diff
@@ -4,6 +4,10 @@
  Metrics for scalability measurement
 =====================================
 
+.. note::
+
+  This feature has been deprecated in |Percona Server| :rn:`5.5.53-38.5`.
```

Comment:
"Users who have installed this plugin but are not using its capability are advised to uninstall the plugin"